build.gradle

```diff
@@ -51,6 +51,7 @@ dependencies {
     compile files('lib/LGoodDatePicker-10.3.1-backport.jar')
 
     compile group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3'
+    compile group: 'org.slf4j', name: 'jcl-over-slf4j', version: '1.7.5'
```

Comment:
As I said previously, this shouldn't be a necessary change. The original code was working (and was more accurate in its scope). It also pollutes the stream of changes with what amounts to a no-op and changes the owner of the change so blame doesn't work as intended. Instead, tidy up the commits to the minimal set of changes.

Reply:
This was added because we want to say explicitly we're using this library instead of just transitively getting it from the google analytics library. Removing this line will do no harm.